_data/applications.yml

```diff
@@ -45,6 +45,16 @@ ostree:
   description: |
     Upgrade OSTree-based systems in your browser.
 
+tukit:
+  title: Software Updates (transactional or tukit)
```

Comment:
The "or" reads a bit confusing here. (e.g. cockpit-ostree are also transactional updates). How about: "tukit transactional software updates"?

Reply:
The software is literally called that https://github.com/openSUSE/transactional-update and tukit is only an intermediary for it like packagekit. I changed the wording a bit, hope that is better.